packages/debug/gdb/package.mk

```diff
@@ -11,6 +11,7 @@ PKG_URL="http://ftpmirror.gnu.org/gdb/$PKG_NAME-$PKG_VERSION.tar.xz"
 PKG_DEPENDS_TARGET="toolchain zlib ncurses expat"
 PKG_LONGDESC="GNU Project debugger, allows you to see what is going on inside another program while it executes."
 # gdb could fail on runtime if build with LTO support
+PKG_BUILD_FLAGS="+size"
```

Comment:
Can we drop the "LTO support" comment - it's a bit out of place as it applied to a previous [`-lto` flag](https://github.com/LibreELEC/LibreELEC.tv/commit/d6ba56ef5f1359587405b33423dd61a7aaae4b41#diff-0cf62576461f566f791be9201c6fbf2b).

Reply:
Done.